pkg/config/config.go

```diff
@@ -90,6 +90,11 @@ const (
 	KeyInclude = "include"
 
 	EnvPusherDebug = "PUSHER_DEBUG"
+
+	BkpManagerMongoURI  = "jobConfigs.cfBackupPlan.target.uri"
```

Comment:
should be KeyBkpManagerMongoURI
KeyGlobalMongoURI and so on following the above convension for value keys